### PR TITLE
Lk star gene full update

### PR DESCRIPTION
**Pull request by ekiernan**
### Description

Changed STARsolo to v2.7.10b for StarSoloFastq task
----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by aawdeh — APPROVED** [review]
LGTM!

**Review by jessicaway — APPROVED** [review]

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 
 # 5.8.3
-2023-06-15 (Date of Last Commit)
+2023-06-23 (Date of Last Commit)
 
+* Updated STARsolo version to v2.7.10b for the StarsoloFastq task
+* Updated STARsolo argument for counting mode to GeneFull_Ex50pAS
```
Comment (kayleemathews) [inline pipelines/skylab/optimus/Optimus.changelog.md:6]: Hey Liz - would you mind updating the docs where the "GeneFull" argument is mentioned? I think it's only mentioned once in the [Optimus Overview doc](https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Optimus_Pipeline/README.md) in the tasks and tools section.

**Review by kayleemathews — CHANGES_REQUESTED** [review]

**Review by kayleemathews — APPROVED** [review]

File: pipelines/skylab/optimus/Optimus.changelog.md (hunk 1 above)
Comment (ekiernan) [reply pipelines/skylab/optimus/Optimus.changelog.md:6]: Thanks @kayleemathews . I updated on this PR: https://github.com/broadinstitute/warp/pull/1010/files

Comment (kayleemathews) [reply pipelines/skylab/optimus/Optimus.changelog.md:6]: Thanks Liz!

**Pull request merged**